web/src/i18n/locales/en.json

```
@@ -134,6 +134,7 @@
         "messageRequired": "A brief message is required",
         "URLValid": "URL must be valid",
         "valid": "Must provide a valid email",
-        "tooManyCharacters": "Please keep the input within {0} characters"
+        "tooManyCharacters": "Please keep the input within {0} characters",
+        "wrongSocialMediaUrl": "This URL is incorrect"
```

Let's differentiate this from the URL valid error. How about:

```suggestion
        "wrongSocialMediaUrl": "This URL doesn't match the expected social network"
```

---

@ann-kilzer same here! (both for english and japanese)